terraformtest_test.go

```diff
@@ -5,16 +5,15 @@ import (
 	"testing"
 )
 
-func TestReadTfPlan(t *testing.T) {
+func TestReadPlanFile(t *testing.T) {
 	t.Parallel()
-
-	wantBiggerThan := 2000
-	tfPlan, err := terraformtest.ReadTfPlan("terraform.tfplan")
+	want := 2000
```

Comment:
We should replace by the current number of bytes of the file being passed to the function, terraform.tfplan in this case.

Reply:
I'll do that in another PR